gridcapa-core-cc-post-processing-app/src/main/java/com/farao_community/farao/core_cc_post_processing/app/CoreCCPostProcessingHandler.java

```diff
@@ -100,9 +98,9 @@ public Set<TaskDto> getAllTaskDtoForBusinessDate(LocalDate localDate) {
                 return new HashSet<>(Arrays.asList(responseEntity.getBody()));
             }
         } catch (Exception e) {
-            LOGGER.error("Error during automatic launch", e);
+            LOGGER.error("Exception during retrieval of the tasks status for date {}", localDate, e);
```

Comment:
I don't understand why you talk about "task status" in this method. Don't we retrieve the full task? All I see in this method that could refer to a status is the HttpStatus from the Http request...